src/scrapers/visa-cal.ts

```diff
@@ -353,6 +357,7 @@ class VisaCalScraper extends BaseScraperWithBrowser<ScraperSpecificCredentials>
   }
 
   getLoginOptions(credentials: ScraperSpecificCredentials): LoginOptions {
+    this.requestPromise = this.page.waitForRequest(SSO_AUTHORIZATION_REQUEST_ENDPOINT);
```

Comment:
Considre adding a short timeout to support the old method of getting the token form local storage

```suggestion
    this.requestPromise = this.page
        .waitForRequest(SSO_AUTHORIZATION_REQUEST_ENDPOINT, { timeout: 10_000 })
        .catch(e => {
            debug("error while waiting for the token request", e);
            return undefined;
        });
```